### PR TITLE
Preliminary GB2018-RW

### PR DESCRIPTION
**Pull request by robin-wang**
Combed through codes, with several points to raise and discuss. See issue for details.

### CODE REVIEW

**Comment by robin-wang** [comment]
#14

File: 2_delivery_care.do [hunk 1]
```diff
@@ -185,9 +192,6 @@ c_sba_eff2_q
 			if inlist(country_name,"CostaRica2018") {	
 				global mn19 "mn19a mn19b mn19i"
 			} 
-			if inlist(country_name,"Turkmenistan2019") {
```
Comment (wengxyu1030) [inline 2_delivery_care.do:188]: Why it's deleted?

File: MICS6_Master.do [hunk 2]
```diff
@@ -120,7 +120,8 @@ foreach name in $newMICS6countries {
 
 * Housekeeping
 	drop country_name
-	
+			pause on
```
Comment (wengxyu1030) [inline MICS6_Master.do:123]: Could you please clarify why we need the pause on command here?

File: MICS6_Master.do [hunk 3]
```diff
@@ -120,7 +120,8 @@ foreach name in $newMICS6countries {
...
+		pause RW itn
```
Comment (wengxyu1030) [inline MICS6_Master.do:124]: Could you please clarify why we need the pause on command here?

File: MICS6_Master.do (hunk 2 above)
Comment (wengxyu1030) [inline MICS6_Master.do:123]: Could you clarify why we have the pause command here?

**Review by wengxyu1030 — COMMENTED** [review]
I have a small question about the code (please see the comment at the specific line). Other than that it looks good to go.

File: MICS6_Master.do (hunk 3 above)
Comment (robin-wang) [reply MICS6_Master.do]: Corrected. please see new merge.

**Pull request merged**